astropy/nddata/utils.py

```diff
@@ -519,9 +519,10 @@ def __init__(self, data, position, size, wcs=None, mode='trim',
             units of pixels.  ``size`` can also be a
             `~astropy.units.Quantity` object or contain
             `~astropy.units.Quantity` objects.  Such
-            `~astropy.units.Quantity` objects must be in pixel or angular
-            units.  See the ``mode`` keyword for additional details on
-            the final cutout size.
+            `~astropy.units.Quantity` objects must be in pixel or
+            angular units.  If ``size`` contains any floats, they will
```

Comment:
I think what is confusing is that it currently says that any sizes will be converted to integers, but what you really mean is that `size` will be converted to an integer number of pixels, rounding to the nearest pixel.

When I first read the line last night (and again this morning) I took the current phrasing to mean that a size in angular units would, if it was a float, be rounded to the nearest integer...that may be an idiosyncratic reading, so revise or not as you think is appropriate :)